front/ui/ui-core/src/components/inputs/datePicker/useDatePicker.ts

```diff
@@ -68,19 +70,21 @@ export default function useDatePicker(datePickerProps: DatePickerProps) {
     if (inputIsDate) {
       const [day, month, year] = newValue.split('/').map(Number);
       const date = new Date(year + 2000, month - 1, day);
+      const isRealDate =
+        !Number.isNaN(date.getTime()) &&
+        date.getFullYear() === year &&
+        date.getMonth() === month - 1 &&
+        date.getDate() === day;
```

Comment:
This test is enough :+1: 
```suggestion
      const isRealDate = !Number.isNaN(date.getTime());
```

Reply:
Done